src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.Http2.cs

```diff
@@ -1526,6 +1528,9 @@ public async Task Dispose_ProcessingResponse_OK()
                     } while (readLength != 0);
                 }
 
+                // Wait until server finished processing
+                // so that totalSent is fully updated
+                serverProcessing.Wait();
```

Comment:
We shouldn't be using synchronous `.Wait()` here. Instead, this should be `await serverProcessing;`

Reply:
Oh...never mind. This is not a task. :)

Reply:
Note: this is not a Task, it's MRE. I'm getting compile errors with that:

```
HttpClientHandlerTest.Http2.cs(1533,17): error CS0012: The type 'IAsyncOperationWithProgress<,>' is defined in an assembly that is not referenced. You must add a reference to assembly 'Windows, Version=255.255.255.255, Culture=neutral,
PublicKeyToken=null, ContentType=WindowsRuntime'. [F:\src\corefx\src\System.Net.Http\tests\FunctionalTests\System.Net.Http.Functional.Tests.csproj]
```

Reply:
unless you mean await Task.Run(() => serverProcessing.Wait());

Reply:
added the timeout, let me know if you want me to change to Task.Run as well